tutorials/katacoda/thanos/4-receiver-agent/step2.md

```diff
@@ -0,0 +1,100 @@
+
+# Setup `batmobile` and `batcopter` lightweight metric collection
+
+With `Wayne Enterprise` platform ready to ingest Remote Write data, we need to think about how we satisfy our three requirements:
+
+1. Implement a global view of this data. `Wayne Enterprises` needs to know what is happening in all company parts - including secret ones!
+2. `Batmobile` and `Batcopter` can be out of network for some duration of time. You don't want to lose precious data.
+3. `Batmobile` and `Batcopter` do not have large compute power, so you want an efficient solution that avoids extra computations and storage.
+
+How are we going to do this?
+
+Previously our recommended option was to deploy Prometheus on our edge place that scrapes required applications, and remote writes all metrics to Thanos Receive. This will work great and satisfy 1st and 2nd requirements, but there are a couple of things that Prometheus offers that we don't need in this particular scenario:
```

Comment:
```suggestion
Previously, the recommended approach was to deploy Prometheus to our edge environment which scrapes required applications, then remote writes all metrics to Thanos Receive. This will work and satisfy 1st and 2nd requirements, however Prometheus does some things that we don't need in this specific scenario:
```